WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -176,6 +166,7 @@ public void showStatsActivityForSite(int localTableSiteId, NoteBlockRangeType ra
             Intent intent = new Intent(this, StatsViewAllActivity.class);
```

Comment:
Instead of creating the intent here, can this be moved to an [ActivityLauncher](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java) routine?